### ISSUE TITLE
user-select: contain does not require the -webkit and -moz prefixes

### ISSUE THREAD

**Issue by yisibl** [postcss/autoprefixer#1506]
-webkit and -moz are wrong, these browsers don't implement this value.

```css
/*
* Prefixed by https://autoprefixer.github.io
* PostCSS: v8.4.14,
* Autoprefixer: v10.4.7
* Browsers: > 0%
*/

.example {
    -webkit-user-select: contain;
       -moz-user-select: contain;
        -ms-user-select: element;
            user-select: contain;
}
```

## Expected

```css
.example {
        -ms-user-select: element;
            user-select: contain;
}
```

**Comment by ai** [comment]
You expect to have

```css
.example {
        -ms-user-select: element;
            user-select: contain;
}
```

Am I right?

**Comment by ai** [comment]
Fixed b0a4007

**Comment by ai** [comment]
Released in 10.4.17.

**Comment by yisibl** [comment]
Thanks!
